clang/test/Sema/unknown-attributes.c

```diff
@@ -0,0 +1,29 @@
+// RUN: %clang_cc1 -fsyntax-only -Wunknown-attribute-namespace=foo,bar -std=c23 -verify %s
```

Comment:
Can you also add a `RUN` line with `-x c++` to show we get the same behavior in C and C++?

Also, we may want to consider adding a `FileCheck` test (instead of a `-verify` test) so that we can validate the correct warning group is emitted after the diagnostic. e.g., unknown namespaces should get you `[-Wunknown-attribute-namespace]` while unknown attributes in a known namespace should still get `[-Wunknown-attributes]`